src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -784,6 +803,7 @@
         default { "0" }
         parmtag { "script_callback" "hou.phm().callback(kwargs)" }
         parmtag { "script_callback_language" "python" }
+        help    "Save the current job bundle to disk."
```

Comment:
Maybe add a second sentence on why someone would want to do this (and simplicity why not). Maybe, "Useful to debug generated jobs without submitting them to Deadline Cloud."